test/spec/PalindromSpec.js

```diff
@@ -9,10 +9,10 @@ describe("Palindrom", function () {
   });
 
   describe("backward compatibility", function () {
-    it("should work if PuppetJs constructor is used", function (done) {
+    it("should work if Puppet constructor is used", function (done) {
       var initSpy = jasmine.createSpy();
 
-      this.palindrom = new PuppetJs({remoteUrl: window.location.href, callback: initSpy});
+      this.palindrom = new Puppet({remoteUrl: window.location.href, callback: initSpy});
```

Comment:
If the link isn't going to the right line you can see here https://github.com/Palindrom/Palindrom/blob/568d554daa38ffea5fa01617f6774a0783bf9269/src/palindrom.js#L799